source/extensions/extensions_build_config.bzl

```diff
@@ -3,342 +3,307 @@ EXTENSIONS = {
     #
     # Access loggers
     #
-
-    "envoy.access_loggers.file":                        "//source/extensions/access_loggers/file:config",
```

Comment:
nit: revert changes to this file. I don't think they are needed anymore.

Reply:
Done. Autoformat! [shakefist]